app/args.py

```diff
@@ -0,0 +1,8 @@
+class Arguments(object):
```

Comment:
@GitSergeyGit base `Argument` class is a good attempt to extract common logic from all the child classes. But there is more to do. First of all the usage interface, `ClParamiko(args).run()` considers all the child classes has `run` method. How about moving this method to some base class with the implementation that will just be raising `NotImplementedError('run')`. This way we will define a contract usage for all child classes, agree?

Reply:
You can leave `Arguments` as is, just introduce another base class that is derived from `Arguments` and used as base class for all the implementations

Reply:
It is important to choose correct name for such a base class, what comes to your mind? Lets have a skype tal with regard to this.

Reply:
I propose to designate a name - CheckFunc or CheckMethod, trite)

Reply:
How about `Shell`?
:)

Reply:
@GitSergeyGit what is the relation between Shell and Arguments? Is it actually correct to derive from both?